testplan/web_ui/testing/src/state/appSlice.js

```diff
@@ -0,0 +1,85 @@
+// @ts-nocheck
+import { createSlice } from '@reduxjs/toolkit/dist/redux-toolkit.esm';
+import { headers as DefaultHeaders } from 'axios/lib/defaults';
+import { flattened } from '../Common/utils';
+
+const __DEV__ = process.env.NODE_ENV !== 'production';
+const API_BASE_URL = process.env.REACT_APP_API_BASE_URL;
```

Comment:
What is the purpose of this env var?

Reply:
because in theory not only MS employees use this. anyway its removed.

Reply:
its back but explained in great detail [here](https://github.com/Morgan-Stanley/testplan/blob/2dd3cd23161070f69e6efdf2b97db5c767c74ad2/testplan/web_ui/testing/.env).